end2end/README.md

```diff
@@ -2,19 +2,111 @@
 
 TODO: Add overview
 
+# LEAF Automated Tests
```

Comment:
Note instructions have been updated in: https://github.com/department-of-veterans-affairs/LEAF-Automated-Tests/pull/9